src/api/driftApi.ts

```diff
@@ -1,3 +1,5 @@
+import { GRegulering } from '~types/GRegulering';
```

Comment:
Mest fordi det ser rart ut med `GR` først.. gjør det noe om vi kaller alle tingene `Regulering` i stedet for `GRegulering`? En potensiell added benefit er at vi kanskje kan ha andre typer regulering senere, eksempelvis regulering av sats-multiplikatoren (2,28/2,48).